src/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
```

Comment:
Usually we add imports on top of css file, it's strange that you using font that is not imported yet